string class_id
uint32 class_type
float64 conf
uint16[] indexes_x
uint16[] indexes_y
